src/components/AddNEditForm/index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable indent */
+import React from 'react';
+
+import { Grid, Input, Typography } from '@material-ui/core';
+
+import Button from '../Button/Button';
+import MultipleSelect from '../MultiSelect';
+import NumberInput from '../NumberInput';
+import SelectTag from '../Select';
+import { StyledMainContainerGrid } from './style';
+
+const AddNEditForm = ({ fields }) => {
```

Comment:
Rename it to AddAndEdit Form

Reply:
And I think if this is common Form use some appropriate naming for it instead of using AddAndEdit form ..
Like CommonGridBasedForm